test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateMaterialX.py

```diff
@@ -265,7 +265,11 @@ def testOCIOIntegration(self):
         """Test that we can color manage using Maya OCIO fragments."""
         cmds.file(new=True, force=True)
         # This config has file rules for all the new textures:
-        configFile = testUtils.getTestScene("MaterialX", "studio-config-v1.0.0_aces-v1.3_ocio-v2.0.ocio")
+        if (Usd.GetVersion() >= (0, 23, 11)):
+            # USD starting at 23.11 we no longer requires file rules to get OCIO results. Metadata will be usable.
+            configFile = testUtils.getTestScene("MaterialX", "no-rule-studio-config-v1.0.0_aces-v1.3_ocio-v2.0.ocio")
```

Comment:
USD 23.11 no longer requires file rules to get proper colorspace for a file if the file has colorspace metadata.